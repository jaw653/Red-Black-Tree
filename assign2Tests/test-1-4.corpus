w
u
t
l
u
w
z
n
c
m
k
w
u
t
l
u
w
z
n
c
m
k
