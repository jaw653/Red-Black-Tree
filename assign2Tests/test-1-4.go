s
r
d w
d u
d t
d l
d u
d w
d z
d n
d c
d m
d k
d w
d u
d t
d l
d u
d w
d z
d n
d c
d m
d k
f w
f u
f t
f l
f u
f w
f z
f n
f c
f m
f k
f jabberwock
f " beware   the  
 jabberwock   "
s
r
