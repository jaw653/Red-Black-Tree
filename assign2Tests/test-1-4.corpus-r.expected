0: n-2-B(n-2-B)-
1: l-2-R(n-2-B)-l u-4-R(n-2-B)-r
2: c-2-B(l-2-R)-l =m-2-B(l-2-R)-r =t-2-B(u-4-R)-l w-4-B(u-4-R)-r
3: =k-2-R(c-2-B)-r =z-2-R(w-4-B)-r
Words/Phrases: 22
Nodes: 9
Minimum depth: 3
Maximum depth: 4
Frequency of w: 0
Frequency of u: 0
Frequency of t: 0
Frequency of l: 0
Frequency of u: 0
Frequency of w: 0
Frequency of z: 0
Frequency of n: 0
Frequency of c: 0
Frequency of m: 0
Frequency of k: 0
Frequency of jabberwock: 0
Frequency of beware the jabberwock: 0
EMPTY
Words/Phrases: 0
Nodes: 0
Minimum depth: 0
Maximum depth: 0
